src/main/java/org/gwtbootstrap3/extras/typeahead/client/TypeaheadClientBundle.java

```diff
@@ -0,0 +1,35 @@
+package org.gwtbootstrap3.extras.typeahead.client;
+
+/*
+ * #%L
+ * GwtBootstrap3
+ * %%
+ * Copyright (C) 2013 - 2014 GwtBootstrap3
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.resources.client.ClientBundle;
+import com.google.gwt.resources.client.TextResource;
+
+/**
+ * @author Florian Kremser <florian.kremser@sage.com>
+ */
+public interface TypeaheadClientBundle extends ClientBundle {
+  static final TypeaheadClientBundle INSTANCE = GWT.create(TypeaheadClientBundle.class);
+
+  @ClientBundle.Source("resource/js/typeahead.jquery.min.js")
```

Comment:
Rename file to typeahead.jquery-VERSION.min.cache.js to make the version apparent and leverage gwt caching. Please.